ci-templates/stages.yml

```diff
@@ -175,7 +175,7 @@ stages:
               jdk: '1.12'
           - template: prepare-cache.yaml
 
-      - job: Run_TCKs
+      - job: Run_JDK8_TCKs
```

Comment:
You can drop the underlines here

Reply:
It is following the other job names structure. Should we just name this `RunJDK8TCKs` and the other `RunJDK11TCKs'?